src/assets/wise5/services/ideasSortingService.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable } from '@angular/core';
+
+export type IdeaData = {
+  id: string;
+  text: string;
+  count: number;
+};
+
+@Injectable()
+export class IdeasSortingService {
+  constructor() {}
+
+  sortIdeasByCount(ideas: IdeaData[]): IdeaData[] {
+    return ideas.filter((idea) => idea.count > 0).sort((a, b) => b.count - a.count);
+  }
+
+  sortIdeasById(ideas: IdeaData[]): IdeaData[] {
```

Comment:
"Ideas" is redundant because the class is IdeasSortingService?
```suggestion
  sortById(ideas: IdeaData[]): IdeaData[] {
```